plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -93,6 +93,8 @@ func init() {
 	factory.RegisterFitPredicate("HostName", predicates.PodFitsHost)
 	// Fit is determined by node selector query.
 	factory.RegisterFitPredicate("MatchNodeSelector", predicates.PodSelectorMatches)
+	// Optional, cluster-autoscaler friendly priority function - give used nodes higher priority.
+	factory.RegisterPriorityFunction("MostRequestedPriority", priorities.MostRequestedPriority, 1)
```

Comment:
Right, I understand that this function is not used by default. The thing I don't understand is how we intend to use this function. We can enable it using scheduler config file, but we cannot disable LeastRequestedPriority, because LeastRequestedPriority is a default priority function.

It's not clear to me that this PR is useful without some mechanism to disable LeastRequestedPriority when you enable MostRequestedPriority.